test/Integration/Dialect/TOSA/bf16_floor_v32/bf16_floor.mlir

```diff
@@ -0,0 +1,13 @@
+// REQUIRES: valid_xchess_license
+// RUN: aie-opt %s %tosa-to-linalg% | aie-opt %linalg-to-vector-v32% --convert-vector-to-aievec="aie-target=aieml" -lower-affine | aie-translate -aieml=true --aievec-to-cpp -o dut.cc
+// RUN: xchesscc_wrapper aie2 -f -g +s +w work +o work -I%S -I%aie_runtime_lib%/AIE2 -I %aietools/include -D__AIEARCH__=20 -D__AIENGINE__ -I. %S/testbench.cc dut.cc
+// RUN: mkdir -p data
+// RUN: xca_udm_dbg --aiearch aie-ml -qf -T -P %aietools/data/aie_ml/lib/ -t "%S/../profiling.tcl ./work/a.out" >& xca_udm_dbg.stdout
+// RUN: FileCheck --input-file=./xca_udm_dbg.stdout %s
+// CHECK: TEST PASSED
+// Cycle count: 919
```

Comment:
Looks like the v32 doesn't save us much of cycles?

Reply:
Yes, because bfloat16_to_int only support v16bfloat16, we need to extract lower and higher part of v32bfloat16 and convert them to int  separately, at last do concatenation. Only comparison and select ops can save some cycle counts.

Reply:
I see, thanks for the explanation.